.github/workflows/pr-integration.yml

```diff
@@ -124,3 +124,12 @@ jobs:
         author_name: "integrationTest"
         icon_emoji: ":triangular_ruler:"
         fields: job, commit
+
+    - name: Archive WSM logs
```

Comment:
Maybe add a description that the logs are available at the bottom of the screen on the run page, and/or name this `Archive WSM Logs with Github Action -- See Below`

Reply:
LGTM. I don't see Approve on the review widget; maybe they don't show it anymore when you already have enough approving reviewers.